client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -67,12 +94,12 @@ const ProfileInfoEdit = () => {
 
           <Form.Group controlId="formProfilePic">
             <Image
-              src={profilePic}
+              src={profileData.profilePic}
```

Comment:
minor: maybe we can use the `Formik` validation style we're using for other components since we're using a Form here? @pmyip3 @mable-yip @jfan0010 @jxiangfan

Reply:
Can it be refactored later? I need help from @pmyip3 but she has other tasks atm.